src/xcode/ENA/ENA/Source/Scenes/Home/Cells/Statistics/HomeStatisticsCellModel.swift

```diff
@@ -27,6 +27,7 @@ class HomeStatisticsCellModel {
 					.compactMap { cardID in
 						statistics.linkCards.first { $0.header.cardID == cardID }
 					}
+				self?.setupMockLinkCards()
```

Comment:
Can you please move this down to line 47 under #if DEBUG? 

if isUITesting {
        self?.setupMockLinkCards()
}